server/test/test_scanpy_engine.py

```diff
@@ -14,7 +14,8 @@
 class UtilTest(unittest.TestCase):
     def setUp(self):
         args = {'layout': 'umap', 'diffexp': 'ttest', 'max_category_items': 100,
-                'obs_names': None, 'var_names': None, 'diffexp_lfc_cutoff': 0.01}
+                'obs_names': None, 'var_names': None, 'diffexp_lfc_cutoff': 0.01,
+                'nan_to_num': True}
```

Comment:
I checked the build time and adding `nan_to_num: True` doesn't increase the time for the test dataset (I was curious)

Reply:
it does, but the test data set is so tiny that it really isn't something you notice.  Try it on the 10M cell neuron data set for a better test (large X), or on the immune data sets (large annotations).

Reply:
Ah, got it.  Not surprising...